motorway/tests/utils.py

```diff
@@ -5,7 +5,7 @@ class ZMQSockMock(object):
     def __init__(self, control_messages):
         self.control_messages = control_messages
 
-    def send_json(self, message):
+    def send_json(self, message, flags=None):
```

Comment:
Is it some leftover or is it intended? Don't see any usage of this in the PR. :)

Reply:
It used, because the ZMQsocket is mocked in the tests

Reply:
If removed, tests are failing